cmake/Modules/FindAmgXSolver.cmake

```diff
@@ -0,0 +1,74 @@
+# - Try to find AmgX
+# Once done this will define
+#
+#  AMGXSOLVER_FOUND        - system has PETSc
+#  AMGXSOLVER_INCLUDES     - the PETSc include directories
+#  AMGXSOLVER_LIBRARIES    - Link these to use PETSc
+#
+# Redistribution and use is allowed according to the terms of the BSD license.
+# For details see the accompanying COPYING-CMAKE-SCRIPTS file.
```

Comment:
Is this copied from somewhere and hence the BSD-Licence? COPYING-CMAKE-SCRIPTS  is not in this PR and probably not applies to other files. Comment seems wrong and needs an update.